src/data/integration.ts

```diff
@@ -36,6 +39,24 @@ export interface IntegrationSetup {
   seconds?: number;
 }
 
+export interface IntegrationLogInfo {
+  domain: string;
+  level?: number;
+}
+
+export enum LogSeverity {
+  CRITICAL = 50,
+  FATAL = 50,
```

Comment:
Why 2 names with the same value?  New rules in #17299 throw an error for this, and doesn't look like we use either.

Reply:
[Python changed the name but preserved backcompat](https://github.com/python/cpython/blob/263d8aa017bd6d858aa9488154bc160fa7249a9e/Lib/logging/__init__.py#L92)

Reply:
But there's no reason we need to keep fatal in the `enum`, right?

Reply:
I think its fine to drop it